cards/eye-ddx/card.md

```diff
@@ -0,0 +1,33 @@
+---
+
+title: Rule of 2s for Ocular Emergencies
+author:
+    - Puja Gopal, MD
+created: 2017/04/14
+updates: null
+categories:
+    - Ophthalmology
+ 
+---
+# Background
+
+Eye complaints are a common chief complaint in the emergency department (ED). Approximately 3.3% of patient visits to the ED were related to an eye or ear complaint in 2005. Having a systematic approach to ocular complaints is especially important as each structure of the eye holds within it different pathology. It is often difficult to remember individual diagnoses for various complaints such as red eye, painful eye, or vision loss, especially during busy shifts. The rule of 2s, as illustrated below, is a simple method with an anatomical approach to major ocular diagnoses encountered in the ED. It ensures one doesn't miss clinically significant diagnoses and requires only a basic understanding of the anatomy of the eye. The rule of 2s is easy to remember as it contains the number 2 in the context of eyes and proceeds from layer to layer of the eye in an anterior to posterior direction highlighting the 2 major associated diagnoses of that layer.  
+This approach is not meant to be all-inclusive of ocular diagnoses but rather is meant to provide a framework one can use to guide the physical exam, such as while using the slit lamp, and to systematically include or eliminate certain conditions.
+
+## Anterior Eye – Utilize slit lamp exam
+![anterior eye picture](image-1.png)

```

Comment:
Can these alt texts be made to be more descriptive? If blind or visually impaired readers read this card they will only be able to "read" `anterior eye picture` when their screen reader software reads the image.